DirectProgramming/DPC++FPGA/ReferenceDesigns/db/src/query11/query11_kernel.cpp

```diff
@@ -160,17 +162,14 @@ bool SubmitQuery11(queue& q, Database& dbinfo, std::string& nation,
   //// Compute Kernel
   auto compute_event = q.single_task<Compute>([=] {
     constexpr int kAccumCacheSize = 15;
-    CachedMemory<DBDecimal,
-                  kPartTableSize,
-                  kAccumCacheSize,
-                  DBIdentifier> partkey_values;
+    fpga_tools::OnchipMemoryWithCache<DBDecimal, kPartTableSize, 
+                                      kAccumCacheSize> partkey_values;
 
     // initialize accumulator
-    partkey_values.Init();
+    partkey_values.init(0);
 
     bool done = false;
 
-    [[intel::initiation_interval(1), intel::ivdep(kAccumCacheSize)]]
```

Comment:
I don't think you should remove the ii1 attribute here?

Reply:
It was not needed to reach II 1 anymore.
Do you think that I should keep it for the compiler to fail if it does not reach II=1?

Reply:
That attribute can serve two purposes.  One is to force the compiler to try harder to achieve II1.  If you're certain that was the only reason it was there in this case, then removing it is fine.  However, the other reason it to guard against future regressions.  I suspect the attribute is useful for this case here, in which case I would leave it in (in both places).